fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -155,14 +155,14 @@ public String submit(MediaType outputFormat, Instant since, List<String> types,
         builder.cosCredentialIbm(properties.get(BulkDataConfigUtil.JOB_PARAMETERS_IBM));
         builder.cosApiKey(properties.get(BulkDataConfigUtil.JOB_PARAMETERS_KEY));
         builder.cosSrvInstId(properties.get(BulkDataConfigUtil.JOB_PARAMETERS_ENDPOINT));
-        
+
         String fhirTenant = FHIRRequestContext.get().getTenantId();
         builder.fhirTenant(fhirTenant);
-        
+
         String fhirDataStoreId = FHIRRequestContext.get().getDataStoreId();
         builder.fhirDataStoreId(fhirDataStoreId);
-        
-        String resourceType = types.get(0);
+
+        String resourceType = String.join(", ", types);
```

Comment:
We use StringJoiner in other parts of the code? any difference to use of String.join vs StringJoiner?

Reply:
String.join is much easy and convenient for the purpose here.